tests/modules/testrdb.c

```diff
@@ -193,6 +193,17 @@ int testrdb_get_before(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     return REDISMODULE_OK;
 }
 
+int testrdb_async_loading_get_before(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
+{
+    REDISMODULE_NOT_USED(argv);
+    if (argc != 1){
+        RedisModule_WrongArity(ctx);
+        return REDISMODULE_OK;
+    }
+    RedisModule_ReplyWithLongLong(ctx, async_loading);
```

Comment:
i meant to return the value of `before_str_temp`, so we don't just wait for the notification that async loading started, but also make sure the REDISMODULE_AUX_BEFORE_RDB was called.

Reply:
let's also put some comment above the function to mention it's a testing hack to control the timing of the test, and not something a valid module would do.

Reply:
Sure thing, done